op-node/flags/flags.go

```diff
@@ -73,6 +73,12 @@ var (
 		EnvVars:  prefixEnvVars("L1_BEACON"),
 		Category: RollupCategory,
 	}
+	SupervisorAddr = &cli.StringFlag{
+		Name:    "supervisor",
+		Usage:   "Experimental feature. Do not use. Address of interop supervisor service for cross-chain safety verification.",
```

Comment:
Not an issue at the moment, but nits on:
* other addresses mention the protocol they use (JSON-RPC).
* Does not need to say "Do not use" unless it's dangerous to do so, which AFAIK it'd be fine to set.